brainiak/fcma/cython_blas.pyx

```diff
@@ -55,7 +55,8 @@ def compute_self_corr_for_voxel_sel(py_trans_a, py_trans_b, py_m, py_n, py_k,
     py_alpha: float
     the weight applied to the first matrix A
 
-    py_a: 2D array in shape [epoch_length, num_voxels] in our case
+    py_a: 2D array
+    In shape [epoch_length, num_voxels] in our case, it is
     the activity data of an epoch, part 1 of the data to be correlated with.
```

Comment:
This still does not read like a sentence:
"In shape [epoch_length, num_voxels] in our case, it is the activity data of an epoch, part 1 of the data to be correlated with."
Also, we usually put the shape in the type line.